lib/disability_compensation/providers/claims_service/lighthouse_claims_service_provider.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'disability_compensation/providers/intent_to_file/intent_to_file_provider'
 require 'disability_compensation/responses/intent_to_files_response'
```

Comment:
however, why the intent to file provider and intent to file response are in this file is a mystery.
can we clean this up, please? 🙏